pkgs/development/python-modules/sphinx-multitoc-numbering/default.nix

```diff
@@ -2,25 +2,38 @@
 , buildPythonPackage
 , fetchPypi
 , sphinx
+, pythonOlder
+, setuptools
 }:
 
 buildPythonPackage rec {
   pname = "sphinx-multitoc-numbering";
   version = "0.1.3";
-  format = "pyproject";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "c9607671ac511236fa5d61a7491c1031e700e8d498c9d2418e6c61d1251209ae";
+    hash = "sha256-yWB2caxREjb6XWGnSRwQMecA6NSYydJBjmxh0SUSCa4=";
   };
 
-  propagatedBuildInputs = [ sphinx ];
+  nativeBuildInputs = [
+    setuptools
+  ];
+
+  propagatedBuildInputs = [
+    sphinx
```

Comment:
I think this is a general note but if we propagate sphinx, it could become pretty hard to overwrite it. Maybe we need to do something similar like we already do with pytest plugins and only putting pytest in buildInputs. Or are we already doing that?

Reply:
I'm not sure if it's already done but it would make sense to go the same route.